sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -432,6 +432,7 @@ singleFieldRelevanceFunctionName
 multiFieldRelevanceFunctionName
     : MULTI_MATCH
     | SIMPLE_QUERY_STRING
+    | SIMPLEQUERYSTRING
```

Comment:
Legacy engine never had such syntax (with square brackets `[]`): `func(['field']...)`

Reply:
This is correct.  For whatever reason, it doesn't appear at that simplequerystring was included.  

We should just close this PR without merging... I don't think it's needed.